src/CodeCoverageOptions.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+/**
+ * This file is part of the friends-of-phpspec/phpspec-code-coverage package.
+ *
+ * @author ek9 <dev@ek9.co>
+ * @license MIT
+ *
+ * For the full copyright and license information, please see the LICENSE file
+ * that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace FriendsOfPhpSpec\PhpSpec\CodeCoverage;
+
+class CodeCoverageOptions
+{
+    /**
+     * @var array<string, mixed>
+     */
+    private $options;
+
+    /**
+     * @param array<string, mixed> $options
```

Comment:
Since the list of option is fixed, maybe it can be interesting to describe the array shape like this: https://phpstan.org/writing-php-code/phpdoc-types#array-shapes ?

Reply:
it would definitely be a nice addition. However, the same problem is true also here, as PHPStan array shapes would require to be maintained for different PHPUnit versions, as the library might act on the `format`, `output`, `show_uncovered_files`, `lower_upper_bound`, `high_lower_bound` and `show_only_summary`, but the extension passed through all other supported PHPUnit options as well, as before.

Since PHPStan cannot really distinguish between what PHPUnit versions we're using, it would be a bit overkill for now, IMO.